change/@fluentui-react-accordion-618b038c-c439-4ca2-b065-56fea572a1e6.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: Updates AccordionHeader to use useARIAButtonShorthand over useARIAButton",
+  "packageName": "@fluentui/react-accordion",
+  "email": "bernardo.sunderhus@gmail.com",
+  "dependentChangeType": "none"
```

```suggestion
  "type": "patch",
  "comment": "chore: Updates AccordionHeader to use useARIAButtonShorthand over useARIAButton",
  "packageName": "@fluentui/react-accordion",
  "email": "bernardo.sunderhus@gmail.com",
  "dependentChangeType": "patch"
```

I would say this is a patch since you are actually changing the implementation internally